syncer/checkpoint_test.go

```diff
@@ -123,15 +124,15 @@ func (s *testCheckpointSuite) testGlobalCheckPoint(c *C, cp CheckPoint) {
 	tctx := tcontext.Background()
 
 	// global checkpoint init to min
-	c.Assert(cp.GlobalPoint(), Equals, minCheckpoint)
-	c.Assert(cp.FlushedGlobalPoint(), Equals, minCheckpoint)
+	c.Assert(cp.GlobalPoint(), Equals, minLocation)
```

Comment:
`minLocation` is a function now.

Reply:
fix in e2e705eda2bc1db71795a9eeb7226731e5c64e8d